### PR TITLE
Adding bindings to SQL spans in tracing

### PR DESCRIPTION
**Pull request by summerKK**
I submitted a PR for #801.  😉

### CODE REVIEW

File: src/Sentry/Laravel/Tracing/EventHandler.php [hunk 1]
```diff
@@ -176,6 +184,12 @@ protected function queryExecutedHandler(DatabaseEvents\QueryExecuted $query): vo
             }
         }
 
+        if ($this->recordSqlBindings) {
+            $context->setData(array_merge($context->getData(), [
+                'db.sql.bindings' => $query->bindings
+            ]));
+        }
```
Comment (cleptric) [inline src/Sentry/Laravel/Tracing/EventHandler.php:191]: Can you move this above `traceSqlQueryOrigins`, so we have the same order?

File: src/Sentry/Laravel/Tracing/EventHandler.php [hunk 2]
```diff
@@ -83,6 +90,7 @@ public function __construct(array $config)
     {
         $this->traceSqlQueries = ($config['sql_queries'] ?? true) === true;
         $this->traceSqlQueryOrigins = ($config['sql_origin'] ?? true) === true;
+        $this->recordSqlBindings = ($config['sql_bindings'] ?? true) === true;
```
Comment (cleptric) [inline src/Sentry/Laravel/Tracing/EventHandler.php:93]: Can you move this below `$this->traceSqlQueries`?

File: src/Sentry/Laravel/Tracing/EventHandler.php [hunk 3]
```diff
@@ -176,6 +184,12 @@ protected function queryExecutedHandler(DatabaseEvents\QueryExecuted $query): vo
             }
         }
 
+        if ($this->recordSqlBindings) {
+            $context->setData(array_merge($context->getData(), [
+                'db.sql.bindings' => $query->bindings
```
Comment (cleptric) [inline src/Sentry/Laravel/Tracing/EventHandler.php:189]: @AbhiPrasad any better ideas for the naming here?

**Review by cleptric — COMMENTED** [review]
Thank you, left some super nitpicky comments 😄

File: src/Sentry/Laravel/Tracing/EventHandler.php (hunk 1 above)
Comment (stayallive) [reply src/Sentry/Laravel/Tracing/EventHandler.php]: I've gone ahead and made those changes 👍

File: src/Sentry/Laravel/Tracing/EventHandler.php (hunk 2 above)
Comment (stayallive) [reply src/Sentry/Laravel/Tracing/EventHandler.php]: I've gone ahead and made those changes 👍

**Review by stayallive — APPROVED** [review]
I've cleaned up the tests a bit since we already had some stuff in place for testing queries. But otherwise looking good, thanks!

Let's see if we need to adjust the `db.sql.bindings` key but otherwise 🚢

File: src/Sentry/Laravel/Tracing/EventHandler.php (hunk 3 above)
Comment (AbhiPrasad) [reply src/Sentry/Laravel/Tracing/EventHandler.php]: Not solved in opentelemetry unfortunately - https://github.com/open-telemetry/opentelemetry-specification/pull/3092

The JS otel sdk has `db.postgresql.values` https://github.com/epsagon/opentelemetry-js-contrib/blob/1f6299d9d48ef2f7b206170827d3858e947474db/plugins/node/opentelemetry-instrumentation-pg/src/enums/AttributeNames.ts#L18

I would be open to `db.sql.values`, but that does make this more general, maybe the constrained definition of `db.sql.bindings` is good here.

**Pull request merged**